### PR TITLE
Homework6

### PR DESCRIPTION
**Pull request by Karl5236543**

### CODE REVIEW

File: homework6.py [hunk 1]
```diff
@@ -0,0 +1,194 @@
+import json
```
Comment (katevoskresenska) [inline homework6.py:1]: Remove files that doesn't relate to HW Classes

File: homework6.py [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+            raise TypeError("ip-адресс должен быть строкой")
+        actet_list = ip.split(Ipv4_manager._sep)
+        if len(actet_list) != 4:
+            return False
+        if not all(isinstance(actet, int) for actet in actet_list):
+            return False
+        parsed_actet_list = [int(actet) for actet in actet_list]
+        if not all(0 <= actet <= 255 for actet in parsed_actet_list):
+            return False
+        return True
```
Comment (katevoskresenska) [inline homework6.py:44]: cool

File: homework6.py [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+        with open(path, 'w') as f:
+            json.dump(data, f)
+    
+    @staticmethod
+    def load(path):
+        if not os.path.exists(path):
+            raise FileNotFoundError("файл не найден")
+        with open(path) as f:
+            return json.load(f)
+
```
Comment (katevoskresenska) [inline homework6.py:59]: It would be nice to add one more method for JSON validation to catch JSONDecodeError

File: homework5.py [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+#в сутках длиной один час в целом для сайта.
+import csv
+from datetime import *
+
+#----------------------------------------------------------------------------------#
+# 1
+#----------------------------------------------------------------------------------#
+def task1():
+    new_lines = []
+    with open(r'.\homework5_data\1.txt', 'r', encoding='utf-8') as f:
```
Comment (mmykhailenko) [inline homework5.py:20]: check if file exists, before open it

**Review by mmykhailenko — APPROVED** [review]

**Pull request merged**